### ISSUE TITLE
OBS AppImage broken

### ISSUE THREAD

**Issue by trevorsandy** [AppImage/AppImageKit#628]
As of 2 days ago my AppImage package build is failing with the following error:
```
[ 1469s] Generating squashfs...
[ 1469s] mksquashfs: invalid option
[ 1469s] 
[ 1469s] SYNTAX:mksquashfs source1 source2 ...  dest [options] [-e list of exclude
[ 1469s] dirs/files]
[ 1469s] 
[ 1469s] Filesystem build options:
[ 1469s] -comp <comp>		select <comp> compression
[ 1469s] 			Compressors available:
[ 1469s] 				gzip (default)
[ 1469s] 				lzo
[ 1469s] 				xz
[ 1469s] -b <block_size>		set data block to <block_size>.  Default 128 Kbytes
```
Up to 2 days ago - the build was successful although the behaviour was a bit strange as it would produce a package with my expected version number (e.g. APP_Name-$VERSION-$ARCH.AppImage) only to then delete it and replace with a new package substituting $VERSION with glibc2.14.

I'm building on x86_64 - x86 is broken with unresolvable dependencies.

I haven't rebuilt my Travis package in the past 2 days so the status there remains successful.

**Comment by hideout** [comment]
did you update something on your system?


2018-01-16 11:05 GMT+01:00 Trevor SANDY <notifications@github.com>:

**Comment by trevorsandy** [comment]
Certainly - I'm testing my packages. However, I didn't update anything - to my knowledge - that should break the mksquashfs command. 
Within the AppImage build process, all the components are being built successfully - see [this log](https://gist.github.com/trevorsandy/a95c9f3c47f5c9d8de4b1ab93bd9cd89). And here is [the .yml file](https://github.com/trevorsandy/lpub3d-ci/blob/master/builds/linux/obs/alldeps/appimage.yml).

**Comment by probonopd** [comment]
The location of `mksquashfs` has recently been made configurable: 

https://github.com/AppImage/AppImageKit/blob/5b1927ab424d004e980422db0b29bde5837612a5/src/CMakeLists.txt#L157

Possibly this is causing issues now, resulting the `mksquashfs` from the system being used rather to the private `mksquashfs` on OBS. @TheAssassin, @adrianschroeter can you confirm?

**Comment by adrianschroeter** [comment]
the system mksquashfs was still installed at least due to dependencies .... dropped that now.

Will check that mksquashfs still can be called.

**Comment by adrianschroeter** [comment]
@trevorsandy should work now again, please test again

**Comment by trevorsandy** [comment]
@adrianschroeter - Success! Many thanks for your fast correction. Cheers. 👍 🥇 

P.S. - It would be wonderful to have a fix for the broken download page [( #172)](https://github.com/openSUSE/software-o-o/issues/172) sometime soon. :-)
